### ISSUE TITLE
Base url is overwritten

### ISSUE THREAD

**Issue by arkanmgerges** [marcosschroh/python-schema-registry-client#46]
**Describe the bug**
client = SchemaRegistryClient('http://localhost:8082/api/schema-registry/')
Will not work as a base URL,

> REST APIs have a base URL to which the endpoint paths are appended. The base URL is defined by schemes, host, and basePath on the root level of the API specification.
https://swagger.io/docs/specification/2-0/api-host-and-base-path/

I ended to do the following:

from schema_registry.client import SchemaRegistryClient, schema
from schema_registry.client.urls import UrlManager
from schema_registry.serializers import MessageSerializer
from schema_registry.client.paths import paths

```
class CustomSchemaRegistryClient(SchemaRegistryClient):
    def __init__(self, url):
        super().__init__(url)
        relativePaths = [(x, f'.{y}', z) for (x, y, z) in paths]
        self.url_manager = UrlManager(url, relativePaths)


client = CustomSchemaRegistryClient("http://localhost:8082/api/schema-registry/")
```

**To Reproduce**
client = SchemaRegistryClient('http://localhost:8082/api/schema-registry/')
# The base url will be overwritten
schema_id = client.register(subject, avro_schema)

**Expected behavior**
To keep the base url

**Comment by marcosschroh** [comment]
Thanks @arkanmgerges . I will try to fox this. A PR is welcome as well.

**Comment by marcosschroh** [comment]
closed by https://github.com/marcosschroh/python-schema-registry-client/pull/48

**Comment by marcosschroh** [comment]
Released in version [1.2.8](https://pypi.org/project/python-schema-registry-client/1.2.8/)
